mycroft/skills/intent_service.py

```diff
@@ -101,3 +147,54 @@ def handle_detach_skill(self, message):
             p for p in self.engine.intent_parsers if
             not p.name.startswith(skill_name)]
         self.engine.intent_parsers = new_parsers
```

Comment:
i missed this comment, will add the remove functionality, perhaps i should wait on the converse PR since skill name will become skill_id ?